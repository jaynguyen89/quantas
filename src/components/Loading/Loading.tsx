import React from 'react';
import 'src/components/Loading/loading.scss';

const Loading = () => {
    return (
        <span className='loader' />
    );
};

export default Loading;