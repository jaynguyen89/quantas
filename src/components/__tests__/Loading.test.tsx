import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import toJson from 'enzyme-to-json';
import Loading from 'src/components/Loading/Loading';

configure({ adapter: new Adapter() });

describe('Loading.tsx', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Loading />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});