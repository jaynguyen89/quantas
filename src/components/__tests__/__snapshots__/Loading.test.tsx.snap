// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading.tsx renders correctly 1`] = `
<span
  className="loader"
/>
`;
